wiki/Interface/en.md

```diff
@@ -70,9 +70,9 @@ Access this screen by clicking your profile at the top left of the main menu. Yo
 
 ## Song Selection Screen
 
-![](img/Soloplay1.jpg "Song Selection")
+![](img/song-selection.jpg "Song Selection")
```

Comment:
@TPGPL at least to me, the difference between the bottom and the top screenshots wasn't immediately obvious without a caption like on `metadata-comparison.jpg`, mind adding one?

Reply:
i think i will just use a single image instead of a comparison, as its not needed at that exact context + a comparison is already made below